pom.xml

```diff
@@ -535,6 +535,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.ec4j.maven</groupId>
+                <artifactId>editorconfig-maven-plugin</artifactId>
+                <version>0.1.0</version>
+                <configuration>
+                    <excludes>
```

Comment:
Excluded `.bacon/` just in case it's run inside the tee. Excluded `.cache/` for formatter-maven-plugin.

Reply:
is it the same plugin that's used on Jenkins?

Reply:
I don't know anything about Jenkins. This plugin was made by someone who works for Red Hat/Quarkus and I have contributed a few PRs to it.

Reply:
What is the benefit of adding it?

Reply:
It is similar to the formatter-maven-plugin, but at a lower level, and supports other file types besides Java. On Windows, for example, the default IDEA settings are not good. IDEA would allow the wrong EOLs to be created and, in fact, changes every file you edit. Even on Unix, soem editors don't seem to add a newline at end of file.

The `.editorconfig` file automatically tells editors like IDEA to use Unix newlines (and some other settings you can see in that file). The plugin can check if files created outside of an IDE adhere to the format used in the project.

Reply:
I am not completely convinced here - there appears to be some overlap between the two plugins which would increase the maven build time. Given currently the primary platform for Bacon is Linux, what other benefits does adding this plugin give?

Reply:
I tested build time with and without the plugin. I didn't even see a consistent increase in build time taken when it was enabled, sometimes the build was even faster.

This ensures that files are formatted correctly, regardless of IDE used. The indent level of XML files is not dependent on OS at all.

Moreover, I see, e.g., in the pnc codebase with no such editconfig files that someone's editor is creating files without a newline, which is a violation even on Linux. This is required by POSIX https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_403.

Reply:
Ok.